langchain4j/pom.xml

```diff
@@ -23,8 +23,8 @@
         </dependency>
 
         <dependency>
-            <groupId>com.squareup.okhttp3</groupId>
-            <artifactId>okhttp</artifactId>
+            <groupId>com.squareup.retrofit2</groupId>
```

Comment:
Is retrofit required for something in `langchain4j` module? The idea of #1048  was to get rid of all okhttp/retrofit dependencies

Reply:
Yes, fixed ;)